rules/Proj2000.md

```diff
@@ -0,0 +1,27 @@
+# Proj2000: Embed valid resources
+A resource file should be a valid RESX file.
+
+## Compliant
+``` XML
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
How much data should be in the RESX file to be regarded as compliant? Would just a resheader be enough or does it have to be specific values?

Reply:
I've to check explicitly which headers are required. I considered to add that (and the root node name) as a requirement, but for now this should be sufficient.